### ISSUE TITLE
[Automation] broken again on v0.5.5.0

### ISSUE THREAD

**Issue by kamigari84** [ihsoft/TimberbornMods#21]
Or so it seems (both on new-game and load).

Here is the error message:
```
[Error  : Unity Log] TypeLoadException: Failure has occurred while loading a type.
Stack trace:
Automation.Tools.DebugPickTool..ctor () (at <612f9f9928be459dbb8bbe828b30fc57>:0)
System.Reflection.RuntimeConstructorInfo.InternalInvoke (System.Object obj, System.Object[] parameters, System.Boolean wrapExceptions) (at <cb3602b5bf2148caa08db7780ec02b19>:0)
Rethrow as TargetInvocationException: Exception has been thrown by the target of an invocation.
System.Reflection.RuntimeConstructorInfo.InternalInvoke (System.Object obj, System.Object[] parameters, System.Boolean wrapExceptions) (at <cb3602b5bf2148caa08db7780ec02b19>:0)
System.Reflection.RuntimeConstructorInfo.DoInvoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) (at <cb3602b5bf2148caa08db7780ec02b19>:0)
System.Reflection.RuntimeConstructorInfo.Invoke (System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) (at <cb3602b5bf2148caa08db7780ec02b19>:0)
System.Reflection.ConstructorInfo.Invoke (System.Object[] parameters) (at <cb3602b5bf2148caa08db7780ec02b19>:0)
Bindito.Core.Internal.InstanceCreator.CreateUsingEligibleConstructor (System.Type type) (at <77aeff39aa1a4904bcb9a4557d4d140e>:0)
Bindito.Core.Internal.InstanceCreator.CreateInstance (System.Type type) (at <77aeff39aa1a4904bcb9a4557d4d140e>:0)
Bindito.Core.Internal.InstanceProvider.ProvideCreatedInstance (System.Type type) (at <77aeff39aa1a4904bcb9a4557d4d140e>:0)
Bindito.Core.Internal.InstanceProvider.ProvideInstance (Bindito.Core.Internal.ProvisionBinding provisionBinding) (at <77aeff39aa1a4904bcb9a4557d4d140e>:0)
Bindito.Core.Internal.ProviderCreator+<>c__DisplayClass4_0.<CreateProvider>b__0 () (at <77aeff39aa1a4904bcb9a4557d4d140e>:0)
Bindito.Core.Internal.InstanceBank.CreateProviderAndGetInstance (System.Type type, Bindito.Core.Internal.Binding binding) (at <77aeff39aa1a4904bcb9a4557d4d140e>:0)
Bindito.Core.Internal.InstanceBank.TryGetInstance (System.Type type, System.Object& instance) (at <77aeff39aa1a4904bcb9a4557d4d140e>:0)
Bindito.Core.Internal.Container.GetInstance (System.Type type) (at <77aeff39aa1a4904bcb9a4557d4d140e>:0)
Bindito.Core.Internal.Container.GetInstance[T] () (at <77aeff39aa1a4904bcb9a4557d4d140e>:0)
TimberApi.DependencyContainerSystem.DependencyContainer.GetInstance[T] () (at <f7cc79b5c538413f94884f158b136fa0>:0)
Automation.Utils.CustomToolSystem+ToolFactory`1[TTool].Create (TimberApi.ToolSystem.ToolSpecification toolSpecification, Timberborn.ToolSystem.ToolGroup toolGroup) (at <612f9f9928be459dbb8bbe828b30fc57>:0)
TimberApi.ToolSystem.ToolService.LateLoad () (at <f7cc79b5c538413f94884f158b136fa0>:0)
TimberApi.Core.SingletonSystem.SingletonSystemPatcher+<>c.<LoadSingletonsPostfix>b__5_0 (TimberApi.Common.SingletonSystem.ILateLoadableSingleton singleton) (at <2ab3f3caeff947c285d1fdc0fe76c790>:0)
TimberApi.Core.SingletonSystem.SingletonSystemPatcher.LoadSingleton[T] (System.Collections.Generic.IEnumerable`1[T] singletons, System.Action`1[T] action) (at <2ab3f3caeff947c285d1fdc0fe76c790>:0)
TimberApi.Core.SingletonSystem.SingletonSystemPatcher.LoadSingletonsPostfix (Timberborn.SingletonSystem.ISingletonRepository ____singletonRepository) (at <2ab3f3caeff947c285d1fdc0fe76c790>:0)
(wrapper dynamic-method) Timberborn.SingletonSystem.SingletonLifecycleService.DMD<Timberborn.SingletonSystem.SingletonLifecycleService::LoadSingletons>(Timberborn.SingletonSystem.SingletonLifecycleService)
(wrapper dynamic-method) Timberborn.SingletonSystem.SingletonLifecycleService.DMD<Timberborn.SingletonSystem.SingletonLifecycleService::LoadAll>(Timberborn.SingletonSystem.SingletonLifecycleService)
Timberborn.SingletonSystem.SingletonLifecycleAdapter.Start () (at <6e8a35ad066f451bb3cd3d1cd0ec7b12>:0)
```

**Comment by ihsoft** [comment]
Could you please attach the logs?

**Comment by kamigari84** [comment]
Will do that tomorrow ... anyways, that was the only error message

**Comment by kamigari84** [comment]
And just to be sure I disabled every other mod ... still crashed
[Here is the crash logs](https://github.com/ihsoft/TimberbornMods/files/13688278/error-report-2023-12-15-18h46m14s.zip)

**Comment by ihsoft** [comment]
Fixed in `v0.11`.
